configs/pong_no_frameskip_v4/iqnResNet.py

```diff
@@ -0,0 +1,66 @@
+"""Config for DQN on Pong-No_FrameSkip-v4.
```

Comment:
should change filename from `iqnResNet.py` to `iqn_resnet.py`

Reply:
resolved in commit 2cad1a7!!